client/src/modules/fiscal/fiscal.openingBalance.html

```diff
@@ -37,15 +37,18 @@
           <div ng-class="{'text-success': FiscalOBCtrl.balanced, 'text-danger': !FiscalOBCtrl.balanced}">
             <span ng-show="OpeningAccountForm.$submitted" >
               <i class="fa" ng-class="{'fa-check': FiscalOBCtrl.balanced, 'fa-warning': !FiscalOBCtrl.balanced}"></i>
-              <span data-status="balanced" ng-show="FiscalOBCtrl.balanced">{{ 'ACCOUNT.BALANCED' | translate }}</span>
-              <span data-status="not-balanced" ng-show="!FiscalOBCtrl.balanced && FiscalOBCtrl.hasPositive">{{ 'ACCOUNT.NOT_BALANCED' | translate }}</span>
-              <span data-status="not-positive" ng-show="!FiscalOBCtrl.balanced && !FiscalOBCtrl.hasPositive">{{ 'ACCOUNT.NOT_POSITIVE' | translate }}</span>
+              <span data-status="balanced" ng-show="FiscalOBCtrl.balanced" translate>ACCOUNT.BALANCED</span>
+              <span data-status="not-balanced" ng-show="!FiscalOBCtrl.balanced">
```

Comment:
This is cool!

On my machine, I've built the default report dataset.  For Fiscal Year 2017, I did the following:
 1. Click "View Beginning Balance"
 2. Clicked "Import"

And I receive the following screenshot:
![redandgreenmessages](https://user-images.githubusercontent.com/896472/28077995-5ff80dec-6631-11e7-90d2-bdeb9bd3b5fd.png)

It is confusing to me that I get both a success message and an "unbalanced" message.  Which one of these is correct?  What is the intended interpretation?  Should I be able to click the "Import" button?